Calculator/Calculator/Model/Formula.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Formula.swift
+//  Calculator
+//
+//  Created by Danny on 2/11/24.
+//
+
+import Foundation
+
+struct Formula {
+    var operands: CalculatorItemQueue<Double>
+    var operators: CalculatorItemQueue<Operator>
+    let defaultValue: Double = 0
+    
+    mutating func result() -> Double {
+        var valueOfResult: Double = operands.dequeue() ?? defaultValue
+        
+        while !operands.isEmpty {
+            if !operators.isEmpty{
```

Comment:
저는 개인적으로 명확하게 조건을 표시하는 편입니다 ㅎㅎ
```suggestion
        while operands.isEmpty == false {
            if operators.isEmpty == false {
```

Reply:
조건을 명확하게 표시해주는 것이 코드를 확인할 때 더 명확하게 확인할 수 있는 것 같습니다. 저도 명확하게 조건을 표시를 하는 쪽으로 연습해야겠습니다.!! 😆